purpleair-aqi.js

```diff
@@ -206,24 +200,23 @@ function haversine(start, end) {
  */
 
 async function getSensorData(sensorId) {
-
   const sensorCache = `sensor-${sensorId}-data.json`;
-     var req = new Request(`${API_URL}/v1/sensors/${sensorId}`);
-     req.headers = {"X-API-Key": API_key} ;
+  var req = new Request(`${API_URL}/v1/sensors/${sensorId}`);
+  req.headers = {"X-API-Key": API_KEY} ;
 
   let json = await req.loadJSON();
 
   try {
     // Check that our results are what we expect
     if (json && json.sensor) {
       console.log(`Sensor data looks good, will cache.`);
-      const sensorData = { json, last_seen: Date.now() }
+      const sensorData = { json, updatedAt: Date.now() }
```

Comment:
If you want to rename this to `last_seen` we'd need to change all of the `updatedAt`s, as this is just the local cache timestamp name.